doc/spectral.rst

```diff
@@ -0,0 +1,137 @@
+.. _spectral:
+
+Spectral (FFT) analysis
+-----------------------
+
+.. ipython:: python
+   :suppress:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import xarray as xr
+    import xrscipy.signal as dsp
```

Comment:
Probably we can remove them?

Reply:
it's a question of convenience, many of use used `xarray_dsp` under the `dsp` alias for convenience in interactive sessions (filtering signals, spectral analysis during shot days).
Perhaps this is connected to teh question of creatign somethign like `xarray.api` (see below)